src/Utilities.php

```diff
@@ -57,6 +57,10 @@ public static function unescape($value)
      */
     public static function binarySidToString($value)
     {
+        if (empty($value)) {
```

Comment:
If this is checking for empty string then it should be `if ($value === '')`, and if it's also checking for null then it can be `empty()` but the docblock needs updating. I suspect the former is right?

Reply:
(Not that I'm a maintainer here!)